features/steps/solvers.py

```diff
@@ -47,8 +40,48 @@ def step_impl(context):
 @then("they should include at least the minimum set of solvers")
 def step_impl(context):
     """Verify all requested solvers are available."""
-    print(context.result)
-    requested_solvers = context.result["requested_solvers"]
     available_solvers = context.result["available_solvers"]
-    for r_s in requested_solvers:
-        assert_that(available_solvers, has_item(r_s))
+    for a_s in available_solvers:
+        assert re.match("solver-(rhel|fedora)-.*\d-py3\d", a_s)  # noqa: W605
+
+
+@then("schedule solver analyses for package {package_name} with version {package_version}")
+def step_impl(context, package_name, package_version):
+    """Schedule solver jobs for all available solvers."""
+    url = f"{context.scheme}://{context.management_api_host}/api/v1/solver/python?secret=secret101"
```

Comment:
```suggestion
    url = f"{context.scheme}://{context.management_api_host}/api/v1/solver/python?secret={os.getenv['THOTH_MANAGEMENT_API_SECRET']}"
```
Please do not hardcode secret.

Reply:
sorry, it leftover, I'll delete, thanks